joomla/joomla-cms
# Fixed user field default value issue

#### Summary of Changes

Simple fix in libraries/cms/form/field/user.php
#### Testing Instructions

Try to use the JFormFieldUser in an XML form with the default value set to "current". The name is displayed correctly in the text input but the hidden input has a value of 0. When this fix is applied the hidden field should have the value of the currently logged in user.
